pythonbrasil-12/lightning_talks/README.md

```diff
@@ -0,0 +1,2 @@
+### [Python Brasil [12]](/pythonbrasil-12/README.md)
+- [SciPYLA and SciPySP](scipyla_scipysp)
```

Comment:
Pois... sorry!
Primeira vez usando git, to tentando me virar sem help aqui ... rs
Eu achei que estava linkando o diretório onde teriam os slides, mas não sei como fazer para enviar o PDF para o diretório antes do merge.

Se alguém quiser me ajudar, to no telegram :-)